packages/cli/src/userInterface.ts

```diff
@@ -151,19 +148,35 @@ export class DialogManager implements Dialog {
     return undefined;
   }
 
-  private showMessage(msg: IMessage) {
-    switch (msg.level) {
-      case MsgLevel.Info:
-        CLILogProvider.necessaryLog(LogLevel.Info, msg.description);
-        break;
-      case MsgLevel.Warning:
-        CLILogProvider.necessaryLog(LogLevel.Warning, msg.description);
-        break;
-      case MsgLevel.Error:
-        CLILogProvider.necessaryLog(LogLevel.Error, msg.description);
-        break;
+  private async showMessage(msg: IMessage): Promise<string | undefined> {
+    if (msg.items && msg.items.length > 0) {
+      const answers = await inquirer.prompt([{
+        name: DialogType.Show,
+        type: "list",
+        message: msg.description,
+        choices: msg.items
+      }]);
+      if (DialogType.Show in answers) {
+        return answers[DialogType.Show];
+      }
+      else {
+        throw InquirerAnswerNotFound(msg);
```

Comment:
Where do we capture this error?

Reply:
I think cli upper level(probably the runcommand level) will catch that?
and also I am a bit unsure about whether we need to handle the error, since it seems impossible for the list question returns an answer not in the list.